### PR TITLE
Removed internalization of one-time-use pkgs

### LINKED ISSUE #19: Don't internalize and push one-time use packages to Nexus (speed enhancement)
Downloading, internalizing, and pushing one-time use packages is currently taking up quite a bit of setup time.

We need to explore the possibility of cutting down the amount of packages we internalize and push, especially if they will not be reused again.

In initial testing, this can potentially halve the total setup time, from 1 hour to 30 minutes.

### PR DESCRIPTION
**Pull request by adilio**
Closes #19 .

ThisPR removes code that internalizes one-time-use packages, as they won't be used again and slow down initial setup. This will greatly improve speed of total setup.

### CODE REVIEW

File: Start-C4BSetup.ps1 [hunk 1]
```diff
@@ -108,28 +108,20 @@ Write-Host "Downloading nupkg files to C:\choco-setup\packages." -ForegroundColo
...
-    Foreach-Object {
-	    choco download $_ --no-progress --force --internalize --internalize-all-urls --append-use-original-location --source="'https://community.chocolatey.org/api/v2/'" --output-directory $PkgsDir
-    }
-
-# We must use the 2.2.7 versions of these packages, so we need to download/internalize these specific items
-@('aspnetcore-runtimepackagestore','dotnetcore-windowshosting') |
-    Foreach-Object {
-	    choco download $_ --version 2.2.7 --no-progress --force --internalize --internalize-all-urls --append-use-original-location --source="'https://community.chocolatey.org/api/v2/'" --output-directory $PkgsDir
-    }
+# Internalize dotnet4.5.2 for ChocolateyGUI (jsut in case)
```
Comment (adilio) [inline Start-C4BSetup.ps1:119]:
```suggestion
# Internalize dotnet4.5.2 for ChocolateyGUI (just in case)
```

Comment (adilio) [inline Start-C4BSetup.ps1:119]:
```suggestion
# Internalize dotnet4.5.2 for ChocolateyGUI (just in case)
```

**Review by adilio — COMMENTED** [review]
need typo fix

**Review by ryanrichter94 — APPROVED** [review]
LGTM 🚀

**Comment by steviecoaster** [comment]
I'm gonna request we pull this back out. Sure it speeds up initial setup, but what happens if they need to reinstall something? Gonna need those dependencies in the repo.

**Comment by ryanrichter94** [comment]
Upon further review and reconsideration. I'm with Stephen that we internalize the CCM packages. We often recommend internalizing the packages from the licensed feed on support calls. It's only right we set good first practice by internalizing these packages and implementing what we preach. 

As a side note to this i'd like to play around with creating a Jenkins job that we ship with this, and possibly any future QDE releases we do, that would internalize the packages from the chocolatey.licensed source, push them to the test repo. And have the job trigger the already existing Update Production Repository Job.

**Comment by adilio** [comment]
So I've just gone over this with @steviecoaster.

I've tested this on Windows Server 2016, 2019, and 2022 multiple times. Leaving the internalization and push to Nexus in the setup adds at least 20 minutes to the install time. If their box is any slower or their  WAN connection slow in any way, this could increase the time by 30 minutes to an hour.

Let's break down what is actually changing. We are leaving out the following packages:

- sql-server-express
- sql-server-management-studio
- dotnet4.6.1

These have dependency on large KB packages, one of which is 2.7GB. These KB's also `no-op` anyway, since they're only relevant to 2012 R2. I see the above as one-time-use only packages, and the benefit of internalizing them only to push them into a repo and then use them immediately after is of very limited utility. If they're somehow unavailable at install, then it's just as likely they won't be available at internalize.

The other packages we're leaving out are Chocolatey's own packages from the licensed feed. These aren't that large, and I'm not opposed to putting them back in based on @ryanrichter94's rationale above. I'll create a separate issue for this, and submit a separate PR.

@steviecoaster & @ryanrichter94 , I've discussed it with you both individually, and we've agreed to leave this PR merged for the time being. We can always revisit later.

Hope that makes sense.

**Pull request merged**